themes/theme-b2b/src/components.ts

```diff
@@ -576,7 +576,7 @@ export const components = {
       stroke: 'gray40',
     },
   },
-  textarea: {
+  textArea: {
```

Comment:
adding `cursor: not-allowed` would be nice. Not you fault but the right time to make it 😀